docs/guide/extension-apis/storage.md

```diff
@@ -139,6 +139,87 @@ const unwatch = showChangelogOnUpdate.watch((newValue) => {
 
 For a full list of properties and methods available, see the [API reference](/api/reference/wxt/storage/interfaces/WxtStorageItem).
 
+### Bulk Operations on Storage Items
+
+When dealing with multiple storage items, you can perform bulk operations to improve performance by reducing the number of individual storage calls. The `storage` API provides several methods to handle bulk operations on defined storage items:
+
+- **`getItemValues`**: Retrieve values of multiple storage items.
+- **`getItemMetas`**: Retrieve metadata for multiple storage items.
+- **`setItemValues`**: Set values for multiple storage items.
+- **`setItemMetas`**: Update metadata for multiple storage items.
+- **`deleteItemValues`**: Remove values (and optionally metadata) of multiple storage items.
+
+#### Getting Values of Multiple Storage Items
+
+You can retrieve the values of multiple storage items in a single call using `getItemValues`:
+
+```ts
+const item1 = storage.defineItem<string>('local:item1', {
+  fallback: 'default1',
+});
+const item2 = storage.defineItem<number>('local:item2', { fallback: 0 });
+const item3 = storage.defineItem<boolean>('local:item3', { fallback: false });
+
+const values = await storage.getItemValues({ item1, item2, item3 });
+
+console.log(values);
+// Output:
+// {
+//   item1: 'default1',
+//   item2: 0,
+//   item3: false
+// }
+```
+
+#### Getting Metadata of Multiple Storage Items
+
+Similarly, you can retrieve the metadata of multiple storage items using `getItemMetas`:
+
+```ts
+const metas = await storage.getItemMetas({ item1, item2, item3 });
+
+console.log(metas);
+// Output:
+// {
+//   item1: { ...metadata of item1 },
+//   item2: { ...metadata of item2 },
+//   item3: { ...metadata of item3 }
+// }
+```
+
+#### Setting Values of Multiple Storage Items
+
+You can set values for multiple storage items in a single operation using `setStorageItemValues`:
+
+```ts
+await storage.setItemValues(
+  { item1, item2, item3 },
+  { item1: 'new value', item2: 42, item3: true },
+);
+```
+
+#### Setting Metadata of Multiple Storage Items
+
+To update metadata for multiple storage items, use `setStorageItemMetas`:
+
+```ts
+await storage.setItemMetas(
+  { item1, item2 },
+  {
+    item1: { lastModified: Date.now() },
+    item2: { version: 2 },
+  },
+);
+```
+
+#### Deleting Values of Multiple Storage Items
+
+You can remove the values (and optionally metadata) of multiple storage items efficiently using `deleteItemValues`:
+
+```ts
+await storage.deleteItemValues({ item1, item2 }, { removeMeta: true });
```

Comment:
I forget, is meta not deleted by default?

Reply:
Will have a look

Reply:
Oh, no, looks like I was wrong. We don't delete meta by default: https://github.com/wxt-dev/wxt/blob/4a1bd41df9f26de337e1c9573e5794518be0a726/packages/wxt/src/__tests__/storage.test.ts#L193-L204

Reply:
For this PR, let's leave it as is and update this PR match. I don't recall the exact reason I choose to do that, but I know I chose it for some reason lol.

If we need to change it, it will have to be a breaking change.